### ISSUE TITLE
Applied to a 3D model of the human body as a whole

### ISSUE THREAD

**Issue by LihaoshengQAZ** [Doubiiu/CodeTalker#6]
First of all, thank the author for opening up the code. I am currently confused about this output video. I would like to apply it to a 3D model of the human body as a whole. What method should I use to apply it? Can you provide an answer?

**Comment by Doubiiu** [comment]
Hi~ Thanks for your interest! Let me rephrase your questions as: Now you have a whole 3D model of the human body and you would like to animate the head (face) according to the audio signal using CodeTalker?  If so, the simplest way is to "crop" the head part, and animate it using CodeTalker separately (make sure it is in FLAME or BIWI topology.) I am not sure if this [paper](https://vcai.mpi-inf.mpg.de/projects/3d_speech_driven_gesture/) could be useful for your case.
